app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -54,6 +72,14 @@ def ensure_managers
     network_manager.provider_region = provider_region
   end
 
+  def ensure_block_storage_managers
+    build_block_storage_manager unless block_storage_manager
```

Comment:
I think we can just put this body to ensure_managers. Or calling this from the body of ensure_managers, then we should extract the current into :ensure_network_manager, then add :ensure_block_storage_manager